build/ansible/roles/pmm-images/tasks/main.yml

```diff
@@ -30,6 +27,7 @@
     name:
       - python3-pip
       - rsync
+      - findutils
```

Comment:
It used to be a dependency of some package which we no longer install, so we need to install it explicitly now.